Sprint-2/implement/to-pounds.js

```diff
@@ -4,3 +4,21 @@
 // You will need to declare a function called toPounds with an appropriately named parameter.
 
 // You should call this function a number of times to check it works for different inputs
+function toPounds(penseString) {
+    const penceStringWithoutTrailingP = penseString.slice(
+      0,
+      penseString.length - 1
+    );
```

Comment:
Is there a way you could check that there is a "p" present in the string before we remove the trailing `p`?

At the moment, if you tried doing toPounds("20") it would return "£0.02" instead of "£0.20". 

Perhaps you could try using a different string function to find and replace the "p" character?

Reply:
i have change the function can you please check it ,thank you